src/Lean/Elab/BuiltinNotation.lean

```diff
@@ -493,7 +493,8 @@ private def withLocalIdentFor (stx : Term) (e : Expr) (k : Term → TermElabM Ex
   return DiscrTree.mkNoindexAnnotation e
 
 @[builtin_term_elab «unsafe»]
-def elabUnsafe : TermElab := fun stx expectedType? =>
+def elabUnsafe : TermElab := fun stx expectedType? => do
+  throwErrorIfUnsafe
```

Comment:
I don't think this is actually relevant in the context of prohibiting arbitrary code execution.